application/src/main/java/org/thingsboard/server/service/install/InstallScripts.java

```diff
@@ -212,10 +212,17 @@ public void loadDashboards(TenantId tenantId, CustomerId customerId) throws Exce
     public void loadDemoRuleChains(TenantId tenantId) throws Exception {
         Path ruleChainsDir = Paths.get(getDataDir(), JSON_DIR, DEMO_DIR, RULE_CHAINS_DIR);
         try {
+            JsonNode thermostatRuleChainJson = objectMapper.readTree(ruleChainsDir.resolve("thermostat_rule_chain.json").toFile());
```

Comment:
why not RuleChain savedRuleChain = installScripts.createDefaultRuleChain(getCurrentUser().getTenantId(), request.getName()); ?